pkg/authz/role/service.go

```diff
@@ -93,7 +93,7 @@ func (svc RoleService) List(ctx context.Context, listParams service.ListParams)
 
 	roles, err := svc.Repository.List(ctx, listParams)
 	if err != nil {
-		return roleSpecs, nil
+		return nil, err
```

Comment:
can you still return the empty slice so it's still serialized in the json response?

```
return roleSpecs, err
```